diffrax/_misc.py

```diff
@@ -12,6 +14,10 @@
 from ._custom_types import BoolScalarLike, RealScalarLike
 
 
+# PendingDeprecationWarning are usually supressed by default
+warnings.simplefilter("always", PendingDeprecationWarning)
+
```

Comment:
Ah, this I didn't realise. Let's just use a `DeprecationWarning` then, to avoid having to mutate shared global state.

Reply:
Changed